Jenkinsfile

```diff
@@ -40,22 +40,6 @@ pipeline {
       failFast true
       parallel {
 
-        stage('Lint') {
```

Comment:
❓ - Do we want to get rid of linting here?

Reply:
I can remove this on https://github.com/dcos-labs/ui-kit/pull/655/files 👍